api/src/main/java/javax/enterprise/inject/spi/Prioritized.java

```diff
@@ -19,20 +19,30 @@
 
 /**
  * <p>
- * Provides access to the priority value of prioritized elements.
+ * This interface allows some SPI implementation to change their priority programmatically.
  * </p>
  *
+ * <p>
+ * For instance The {@link ObserverMethod} interface extends this interface to set the observer priority.
```

Comment:
For instance, {@link ObserverMethod} extends this interface to define the observer priority.